pkg/server/server.go

```diff
@@ -190,3 +194,39 @@ func (s *Server) Run() error {
 	srv.Addr = s.Addr
 	return srv.ListenAndServe()
 }
+
+// backup performs backup flow.
+func (s *Server) backup(backupDirectoryID int, policyID string) error {
+	ctx := context.Background()
```

Comment:
Can we add function to get the absolute directory in this function now or later ?

Reply:
There's TODO in function body